### PR TITLE
Feature: move bot-lax article xml to /ext

### PR DESCRIPTION
**Pull request by lsh-0**

### CODE REVIEW

File: salt/lax/bot-lax-adaptor.sls [hunk 1]
```diff
@@ -119,19 +105,27 @@ dir-{{ path }}:
...
 
+#
+#
+#
+
+move /opt/bot-lax/article-xml/ to /ext/article-xml:
+    cmd.script:
+        - cwd: /ext
+        - source: salt://lax/scripts/mv-article-xml.sh
+        - onlyif:
```
Comment (giorgiosironi) [inline salt/lax/bot-lax-adaptor.sls]: I think we can enable this on all stacks:
https://github.com/elifesciences/builder-private/blob/master/salt/top.sls#L126

Comment (giorgiosironi) [reply salt/lax/bot-lax-adaptor.sls]: That file should create an `/ext` folder even if there's no volume.

Comment (giorgiosironi) [reply salt/lax/bot-lax-adaptor.sls]: (on the default volume)

Comment (lsh-0) [reply salt/lax/bot-lax-adaptor.sls]: PR: https://github.com/elifesciences/builder-private/pull/230

Comment (giorgiosironi) [reply salt/lax/bot-lax-adaptor.sls]: Approved and merged, now `example.top` here should be updated along with losing the `onlyif` statements

**Pull request merged**